_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,136 @@
+---
+name: IFrame has a unique accessible name
+
+description: |
+ Each iframe element has a unique accessible name
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to `iframe` elements within a given [document context](#document-context) that have either `title`, `aria-label` or `aria-labelled-by` attribute.
+
+**Note: This rule does not ensure that the `iframe` targets with the same `src` destination have the same title. It only checks `title` of a given frame for uniqueness across the document context.**
+
+### Expectation
+
+Each test target has a `title`, `aria-label` or `aria-labelledby` attribute value that is unique within the [document context](#document-context) of the element.
```

Comment:
This is pretty vague. What does it mean for a title to be unique in a document context? I think what this is supposed to be is that no test target has the same accessible name, except if they contain the same page? I don't really see why we'd limit this to the document context. Can you clarify?